spec/ruby/core/bignum/negative_spec.rb

```diff
@@ -0,0 +1,9 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe "Bignum#negative?" do
+  it "returns true if self is less than 0" do
```

Comment:
The `x.negative?.should == false` expectations don't exercise what the spec description describes. :confused: